WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -99,7 +100,7 @@ final class OrderListViewController: UIViewController, GhostableViewController {
 
     /// Current top banner that is displayed.
     ///
-    private var topBannerView: TopBannerView?
+    private var topBannerView: UIView?
```

Comment:
I'm not sure to understand this change: 

We're moving from `TopBannerView` to a `UIView` because of the change from `ConstraintsUpdatingHostingController` to `UIHostingController` as we do not need this inside a StackView now: While the former accepts a `View` (so `TopBannerView` worked just fine), the new implementation requires an `UIView`, which `TopBannerView`  doesn't conform to. Is that correct?

Reply:
You almost nailed it @iamgabrielma :) 

Before the property was of type `TopBannerView` because we only could show that kind of view: errors and order feature banners are both rendered via `TopBannerView`. Now we have a different banner, the Upsell Card Readers, that is implemented otherwise (`SwiftUI` view and `UIHostingController`). Because of that, we can make the property more abstract and convert it to UIView.
In general, even if we didn't add the new Upsell Card Readers banner, I would have created the topBannerView property as `UIView`. Making it a `TopBannerView` adds some restraints that we do not actually need, (as you can see no changes were done after making it a UIView). In general, the more abstraction, the better. On the other hand, if we had to cast it to `TopBannerView` somewhere it would have been different.